app/ui-flextab/client/tabs/userInfo.js

```diff
@@ -85,6 +87,15 @@ Template.userInfo.helpers({
 		return userStatus;
 	},
 
+	userStatusText() {
+		if (s.trim(this.statusText)) {
+			return emojione.render(s.escapeHTML(this.statusText));
```

Comment:
I added this code to allow emojis in status messages before I knew how emojis were integrated into Rocket.

Maybe take a look at https://github.com/RocketChat/Rocket.Chat/blob/d53e264b83ccc971236123853e4b68c9eb6beeb4/app/emoji/client/emojiParser.js#L26

Want to make sure custom emojis work!